frontend/projects/valtimo/process-management/src/lib/components/process-management-builder/process-management-builder.component.scss

```
@@ -110,6 +111,273 @@
+    user-select: none;
+
+    &::-webkit-details-marker {
+      display: none;
+    }
+  }
+
+  &__arrow {
+    display: inline-flex;
+    width: 16px;
+    height: 16px;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)